front/components/app/ModelPicker.tsx

```diff
@@ -81,7 +85,7 @@ export default function ModelPicker({
       model.provider_id.length > 0
     ) {
       if (models === null) {
-        refreshModels();
+        refreshModels().catch((e) => console.error(e));
```

Comment:
await is probably fine here?

Reply:
Can't await in useEffect() because useEffect does not take an async function as a parameter.
The standard solution to that problem on stackoverflow and co is to wrap your code in an async block, but then you need to `.catch()` it if you want to comply to the "no dangling promise" rule.
It is useful when you want to have multiple awaits in your `useEffect`. Here is a concrete example:
```
useEffect(() => {
(async() => {
    await one();
    await two();
    await three();
    
  })
})().catch(console.error)
```